README.md

```diff
@@ -440,8 +444,13 @@ These environments must contain the following variables:
 GATSBY_REALM_APP_ID=
 REALM_API_PRIVATE_KEY=
 REALM_API_PUBLIC_KEY=
+GATSBY_REALM_EMPTY_ENV_APP_ID=[The Realm App ID of the empty environment]
```

Comment:
Are these env variables needed for the production and staging workflows? If not, I would suggest separating this into a smaller subsection, describing the empty environment with the new `realm-empty-env.yml` workflow.

Reply:
Changed to use the `empty` environment variables instead of using these `EMPTY_DEV` ones

Reply:
is `GATSBY_REALM_EMPTY_ENV_APP_ID` still needed?

Reply:
No, I just remove them from the README file 👍